<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwUpdaterGuiUi</class>
 <widget class="QDialog" name="AwUpdaterGuiUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Available components</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonSelectAll">
       <property name="text">
        <string>Select all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUnselect">
       <property name="text">
        <string>Unselect</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonInstall">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Install updates</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonInstall</sender>
   <signal>clicked()</signal>
   <receiver>AwUpdaterGuiUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
